jerry-debugger/jerry-client-ws.py

```diff
@@ -728,6 +757,27 @@ def parse_source(debugger, data):
     else:
         logging.debug("No pending breakpoints")
 
+def printSource(self, line_num):
+    last_bp = self.debugger.last_breakpoint_hit
+    lines = last_bp.function.source
+    if last_bp:
+        if last_bp.function.source_name:
+            print("Source: %s" % (last_bp.function.source_name))
+
+        if line_num == 0:
+            start = 0
+            end = len(last_bp.function.source) - 1
+        else:
+            start = max(last_bp.line - line_num, 0)
+            end = min(last_bp.line + line_num-1, len(last_bp.function.source)-1)
+
+        for i in range(start, end):
+            spaces = (4-len(str(i+1)))*" "
+            if i == last_bp.line - 1:
+                print("%d%s>%s" % (i + 1, spaces, lines[i]))
```

Comment:
What about:

    if i == last_bp.line - 1:
      string = " > "
    else
      string = "   "
    print("%4d%s%s" % (i + 1, string, lines[i]))

This way you don't need spaces